_config.yml

```
@@ -38,6 +38,10 @@ pub-pkg: https://pub.dev/packages
 dartpad: https://dartpad.dev
 news: https://news.dartlang.org
 announce: https://groups.google.com/a/dartlang.org/g/announce
+gh: https://github.com
+  dart: https://github.com/dart-lang
```

`dart` can't have an inline value and be a map. Consider something like this instead:

```suggestion
  dart:
    org: https://github.com/dart-lang
```